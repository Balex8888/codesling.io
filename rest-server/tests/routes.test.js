import axios from 'axios';

require('dotenv').config();
require('dotenv').load();

const URL = `${process.env.HOST}:${process.env.PORT}`;

test('rest-server should run and return a 404 to {GET /}', async (done) => {
  let resp = {};
  try {
    resp = await axios.get(URL + '/');
  } catch (e) {
    expect(e.response.status).toBe(404);
  }
  expect(resp).toMatchSnapshot();
  done();
});
