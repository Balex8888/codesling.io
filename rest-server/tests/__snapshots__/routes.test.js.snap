// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rest-server should run and return a 404 to {GET /} 1`] = `Object {}`;
